raw-mm.aliases.lua

```diff
@@ -231,66 +231,19 @@ end
 end
 
   function aconfig2()
-  cecho("<a_darkblue>--<purple>(m&mf) <a_grey>Configuration, continued<a_darkblue>" .. string.rep("-", 48) .. "\n")
-
-  cecho("<a_darkcyan>  Pipe IDs:\n")
-  cecho("<a_darkgrey>    Coltsfoot          Faeleaf            Myrtle            Steam\n")
-
-  cecho(string.format(
-    "%s    ID %s%-16d"
-  .."%sID %s%-16d"
-  .."%sID %s%-15d"
-  .."%sID %s%d\n",
-    "<a_grey>", "<a_cyan>", pipes.coltsfoot.id,
-    "<a_grey>", "<a_cyan>", pipes.faeleaf.id,
-    "<a_grey>", "<a_cyan>", pipes.myrtle.id,
-    "<a_grey>", "<a_cyan>", pipes.steam.id
-  ))
-
-  cecho(string.format(
-    "%s    Puffs %s%-2d%-11s"
-  .."%sPuffs %s%-2d%-11s"
-  .."%sPuffs %s%-2d%-10s"
-  .."%sPuffs %s%-2d\n",
-    "<a_grey>", "<a_cyan>", pipes.coltsfoot.puffs, " ",
-    "<a_grey>", "<a_cyan>", pipes.faeleaf.puffs, " ",
-    "<a_grey>", "<a_cyan>", pipes.myrtle.puffs, " ",
-    "<a_grey>", "<a_cyan>", pipes.steam.puffs
-  ))
+  cecho("<a_darkblue>--<purple>(m&mf) <a_grey>Configuration, MMCONFIG2 continued<a_darkblue>" .. string.rep("-", 48) .. "\n")
+
+  local c1,s1 =
+  unpack(pipes.steam.arty and
+      {"<gold>", "Arty"} or
+          (pipes.steam.lit and {"<a_yellow>", "Lit!"} or {"<a_darkgrey>", "Unlit."})
+  )
+ 
+  -- I know this is ugly but the 'f stringformated hated the versions of it I tried. This works and it can be made pretty some other day.'
+  cecho(f("\n<a_darkcyan>  Steam Pipe: <a_grey>ID " ..pipes.steam.id .."   puffs: " ..pipes.steam.puffs .."   lit? " .. c1 ..s1))
```

Comment:
What issue were you having with this? We should be able to make it work.

Reply:
The issue was that using 'f' to format strings is more particular than using '..var'. and 'f' was choking on out of scope or it was just insisting that 'pipes' was a global.

refactored the code and made a local copy or pipes and sent that.

Below you'll see some of the syntax I tried. It is there just for your eyes and if you wanted to remove it to make the commit cleaner please do.